unmarshal_from_json_map_test.go

```diff
@@ -2301,3 +2301,32 @@ func TestUnmarshalFromJSONMapJSONDataHandler(t *testing.T) {
 		}
 	})
 }
+
+func TestUnmarshalFromJSONMapExcludeKnownFieldsFromMap(t *testing.T) {
+	t.Run("test_exclude_known_fields_from_map_with_empty_map", func(t *testing.T) {
+		p := Person{}
+		result, err := UnmarshalFromJSONMap(map[string]interface{}{"firstName": "string_firstName", "lastName": "string_LastName"}, &p, WithExcludeKnownFieldsFromMap(true))
+		if err != nil {
+			t.Errorf("unexpected error %v", err)
+		}
+		if len(result) != 0 {
+			t.Errorf("failure in excluding untyped fields")
+		}
+	})
+
+	t.Run("test_exclude_known_fields_from_map", func(t *testing.T) {
+		p := Person{}
+		result, err := UnmarshalFromJSONMap(map[string]interface{}{"firstName": "string_firstName", "lastName": "string_LastName", "field_to_exclude": "untyped"}, &p, WithExcludeKnownFieldsFromMap(true))
```

Comment:
"field_to_exclude" here is not excluded because it is not known, so it is rather "field_not_excluded". but maybe it is better to name it "unknown":"string_unknown" to align to the known/unknown fields terminology and examples

Reply:
Agree

Reply:
right, fixed the naming